include/freetds/macros.h

```diff
@@ -70,7 +70,7 @@
 #define TDS_WUR
 #endif
 
-#define TDS_INT2PTR(i) ((void*)(((char*)0)+(i)))
+#define TDS_INT2PTR(i) ((void*)(((char*)0)+((intptr_t)(i))))
```

Comment:
nice, but in some systems intptr_t could be defined by tds_sysdep_public.h which is not included by this file

Reply:
Good point.  Any objection to adding an appropriate `#include` directive?  I see that `tds.h` already includes `tds_sysdep_public.h` ahead of `macros.h`.

Reply:
yes, but some files include `macros.h` without including `tds.h`. I suppose it's safe to include `tds_sysdep_public.h` from `macros.h`